packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -263,6 +264,30 @@ if (import.meta.hot) {
 								path.resolve(resolvedViteConfig.root, clientOutputDirectory)
 							),
 						};
+
+						if (workerConfig.containers?.length) {
+							workerConfig.containers = workerConfig.containers.map(
+								(container) => {
+									const baseDir = workerConfig.configPath
+										? path.dirname(workerConfig.configPath)
+										: resolvedViteConfig.root;
+									// Wrangler's config validation resolves to container.configuration.image, even though it is deprecated
+									const image =
+										container.configuration?.image ?? container.image;
+									if (isDockerfile(image, workerConfig.configPath)) {
```

Comment:
Does it matter that `workerConfig.configPath` is possible `undefined` here? Not sure if it ever can actually be `undefined` or if that's just a types thing.